scripts/build/toolchain/termux_setup_gnu_as_23c.sh

```diff
@@ -0,0 +1,94 @@
+# shellcheck shell=bash disable=SC2034 disable=SC2155
+termux_setup_gnu_as_23c() {
+	if [ "$TERMUX_ON_DEVICE_BUILD" = true ]; then
+		if [ -z "$(command -v as)" ]; then
+			echo "Command 'as' is not installed."
+			if [ "$TERMUX_APP_PACKAGE_MANAGER" = apt ]; then
+				echo "You can install it with"
+				echo
+				echo "  pkg install binutils"
+				echo
+			elif [ "$TERMUX_APP_PACKAGE_MANAGER" = pacman ]; then
+				echo "You can install it with"
+				echo
+				echo "  pacman -S binutils"
+				echo
+			fi
+			exit 1
+		fi
+		return
+	fi
+
+	local GAS_TOOLCHAIN_REVISION="-v0"
+	local GAS_TOOLCHAIN_DIR="$TERMUX_COMMON_CACHEDIR/android-r23c-gas-api-${TERMUX_PKG_API_LEVEL}${GAS_TOOLCHAIN_REVISION}"
+	local NDK="$(dirname "$NDK")"/android-ndk-r23c
+	if [ ! -d "$NDK" ]; then
```

Comment:
this `if` (that I added) is unnecessary, setup-android-sdk.sh already checks if folder exists